markets/models.py

```diff
@@ -166,7 +163,7 @@ def __str__(self) -> str:
         return f"Guessed {self.value} at {self.created_at}"
 
     def get_absolute_url(self) -> str:
-        return reverse("market-results", args=(self.market.slug,))
```

Comment:
i'm not sure this should be changed either tbh, although i could understand if it pointed to the updateview